### ISSUE TITLE
Arcface training strategy(Resnet50, CASIA - Webface, batch size < 128)

### ISSUE THREAD

**Issue by ewigspace1910** [deepinsight/insightface#1950]
im trying to train insightFace on CASIA - Webface, and i tryed to train arcface and cosface from scratch. Due my limit system, i just set batch size to 64. My result when i training arcface, cosface on LFW are 99,3 and 99,4 respectively, not like result in the paper.

could anyone give me some advice on how to train arcface, learning rate...?

Configs  for arc/cosface are same, like this:
## hyper params
weight_decay:  5e-4
logits_scale: 30
logits_margin: 0.5 ( 0.35 for cosface)
drop_ratio: 0.5 
##-------------------
#optimizer
optimizer: SGD
momentum: 0.9
base_lr : 0.01
lr_steps: [20000, 28000] 

Thanks in advance!

**Comment by hoahoa191** [comment]
have you an answer?

**Comment by ewigspace1910** [comment]
@hoahoa191 yeah, i change the logits_scale to 64, base_lr: 0.025, then acc is improved
